Exposure compensation

riggiding opened the pull request:
@bonosoft I'd like to have your OK on this before merging - for the sake of performance I had to do this outside of the existing PhotonLayer pattern.
Also I refactored the button handling into actions and introduced a top menu bar that helps us integrate stuff like the fine support generation more easily.

mwe-ontec committed:
basic erosion algorithm working

mwe-ontec committed:
action based menu, exposure compensation ui

mwe-ontec committed:
added menu bar, exposure correction

mwe-ontec committed:
fixed error when comensation is zero

X3msnake commented:
Tried the JAR from the image-processing branch but it just says loading and nothing happens

cwoac commented:
So, the jar file attached runs, but lacks the menu. However if I export the branch and recompile, the menu appears and all seems to run, so I suspect it's an issue with the Jar.

X3msnake commented:
probably a outdated jar
better to set as a standard to always update the jar so non technical
people can contribute with testing and validating the PR

No dia terça-feira, 5 de maio de 2020, Oliver Matthews <
notifications@github.com> escreveu:

> So, the jar file attached runs, but lacks the menu. However if I export
> the branch and recompile, the menu appears and all seems to run, so I
> suspect it's an issue with the Jar.
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/Photonsters/PhotonFileValidator/pull/64#issuecomment-623995920>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ACUR56WJ44OLLCYG2M5JNKTRP7Y55ANCNFSM4LI54PAA>
> .
>


-- 
Com os melhores cumprimentos,
Vinicius Silva

cwoac committed:
Rebuilt JAR

File changes are from newer (I guess?) intellij rewriting the imports for the generated code differently - I've made no actual changes to them.

cwoac committed:
remove intellij files (sorry)

cwoac commented:
I've rebuilt it from the code and updated the jar file so it can be tested.

X3msnake commented:
thanks Master

No dia terça-feira, 5 de maio de 2020, Oliver Matthews <
notifications@github.com> escreveu:

> I've rebuilt it from the code and updated the jar file so it can be tested.
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/Photonsters/PhotonFileValidator/pull/64#issuecomment-624050928>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ACUR56RESO44H6CQG3URPZLRQAHDHANCNFSM4LI54PAA>
> .
>


-- 
Com os melhores cumprimentos,
Vinicius Silva
